queries/subsquid/ahk/nftListWithSearch.graphql

```diff
@@ -0,0 +1,69 @@
+query tokenListWithSearch(
+  $first: Int!
+  $offset: Int
+  $denyList: [String!]
+  $search: [NFTEntityWhereInput!]
+  $orderBy: [TokenEntityOrderByInput!]
+) {
+  tokenEntities: tokenEntities(
+    orderBy: $orderBy
+    limit: $first
+    offset: $offset
+    where: {
+      nfts_some: {
+        burned_eq: false
+        issuer_not_in: $denyList
+        metadata_not_eq: ""
+        AND: $search
+      }
+    }
+  ) {
+    id
+    blockNumber
+    hash
+    image
+    media
+    name
+    updatedAt
+    createdAt
+    count
+    nfts {
+      id
+      metadata
+      issuer
+      currentOwner
+      blockNumber
+      burned
+      name
+      sn
+      price
+      royalty
+      recipient
```

Comment:
Could you please use something fragments like this? This can avoid enumerating all values.

```
#import "../../fragments/nft.graphql"
#import "../../fragments/nftDetails.graphql"
```

Reply:
done